### ISSUE TITLE
Deactivated Matrix blocks is still visible

### ISSUE THREAD

**Issue by magnusskogen** [verbb/vizy#67]
**Description**
Deactivated Matrix blocks is not hidden in frontend.

**Steps to reproduce**
1. Add a Matrix field to Vizy.
2. Add a block, and deactivate it.
3. View article in any browser. The block is visible, even if it was deactivated.

**Additional info**
- Plugin version: 1.0.5
- Craft version: 3.7.7
- Multi-site: No

**Additional context**

**Comment by magnusskogen** [comment]
Seems like deactivating overall is not working. Deactivated Vizy-blocks appears as well.

**Comment by engram-design** [comment]
I can't seem to replicate the Matrix disabled block behaviour, but disabled Vizy blocks are certainly accessible when querying via `.all()` on the collection. This shouldn't be the case. Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.6](https://github.com/verbb/vizy/releases/tag/1.0.6)
